helper/testhelpers/ldap/ldaphelper.go

```diff
@@ -36,8 +37,9 @@ func PrepareTestContainer(t *testing.T, version string) (cleanup func(), cfg *ld
 	if err = pool.Retry(func() error {
 		logger := hclog.New(nil)
 		client := ldaputil.Client{
-			LDAP:   ldaputil.NewLDAP(),
-			Logger: logger,
+			LDAP:      ldaputil.NewLDAP(),
+			Logger:    logger,
+			DebugBind: true,
```

Comment:
I don't mind keeping DebugBind around, but I think we should disable it in the committed code to avoid the test-race failures (since you've already solved the problem I wanted to use it to debug.)

Reply:
right, good catch